core/templates/services/playthrough.service.ts

```diff
@@ -54,30 +54,31 @@ class CyclicStateTransitionsTracker {
   }
 
   /**
-   * Records learner's transition to a new state into this tracker's path of
-   * visited states.
-   *
-   * If appending the new state would introduce a duplicate state name, then a
-   * cycle has been found. Specifically, if pushing the newest state (N) onto
-   * the path would result in the following pattern:
-   *
-   *    [ ... , N , ... , N ]
-   *
-   * then we update this tracker's latest cycle discovery. The cycle is defined
-   * as the current path of visited states with all states prior to the first
-   * occurrence of N discarded:
-   *
-   *    [ ... // N , ... , N ]  =>  [ N , ... , N ]
-   *
-   * If this *exact* cycle has been discovered before (NOTE: rotations of a
-   * cycle are considered to be different from each other), then we increase the
-   * tracked number of cycle occurrences.
-   * Otherwise, the tracker is completely reset to 1.
-   *
-   * Finally, the path of visited states is reset to a value of [ N ], in hopes
-   * that the exact same cycle is discovered enough times to be considered an
-   * issue.
-   */
+  * Records learner's transition to a new state into this tracker's path of
```

Comment:
Why is this changed? The previous indentation was correct.

Reply:
I did it by mistake .I will fix it